schema/bom-1.4-SNAPSHOT.proto

```diff
@@ -450,3 +456,48 @@ message Evidence {
   repeated LicenseChoice licenses = 1;
   repeated EvidenceCopyright copyright = 2;
 }
+
+enum ReleaseType {
+  // A major release may contain significant changes or may introduce breaking changes.
+  RELEASE_TYPE_MAJOR = 0;
+  // A minor release, also known as an update, may contain a smaller number of changes than major releases.
+  RELEASE_TYPE_MINOR = 1;
+  // Patch releases are typically unplanned and may resolve defects or important security issues.
+  RELEASE_TYPE_PATCH = 2;
+  // A pre-release may include alpha, beta, or release candidates and typically have limited support. They provide the ability to preview a release prior to its general availability.
+  RELEASE_TYPE_PRE_RELEASE = 3;
+  // Internal releases are not for public consumption and are intended to be used exclusively by the project or manufacturer that produced it.
+  RELEASE_TYPE_INTERNAL = 4;
+}
```

Comment:
I like this idea but I'm a lil concerned it's a bit too much of one size fits all. I know versions are messy, and I think the biggest thing I can't see how I'd represent here, is mostly an `epoch` in the linux package world. I'd struggle to think of how we could best represent this beyond like suggestions (USE SEMVER PLZ) but a lil less strict.

Reply:
Resolved in 2e8eca48ba20b431e05e53452c29806bd672e6ec. I've changed the release type from enum to a string, and added descriptions on recommended values.